grunt_tasks/webpack.config.js

```diff
@@ -69,7 +70,14 @@ module.exports = {
             'window.jQuery': 'jquery'
         }),
         // Disable writing the output file if a build error occurs
-        new webpack.NoEmitOnErrorsPlugin()
+        new webpack.NoEmitOnErrorsPlugin(),
+        new GoogleFontsPlugin({
```

Comment:
And, that passing reference is just the css defined in `clients/web/src/stylesheets/layout/global.styl`, not triggered by these changes.